### PR TITLE
Add missing BottomNavigationView badge for retrieved test certificates (EXPOSUREAPP-7851)

### PR DESCRIPTION
**Pull request by d4rken**
Screenshot from 2021-06-16 13-41-58

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/CertificatesViewModel.kt [hunk 1]
```diff
@@ -42,6 +45,18 @@ class CertificatesViewModel @AssistedInject constructor(
...
 
+    val markNewCertsAsSeen = testCertificateRepository.certificates
+        .onEach { wrappers ->
+            wrappers
+                .filter { !it.seenByUser && !it.isCertificateRetrievalPending }
+                .forEach {
+                    testCertificateRepository.markCertificateAsSeenByUser(it.identifier)
+                }
+        }
+        .map { }
```
Comment (BMItter) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/CertificatesViewModel.kt:56]: map { } -> { }

Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/CertificatesViewModel.kt:56]: What do you mean by that?
We just need to be subscribed, we don't want to return a specific value, to achieve that: `.map { Unit }` -> `.map { }` -> `Unit`, is there a better way?

Comment (BMItter) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/CertificatesViewModel.kt:56]: without the unit it seemed to me like a forgotten leftover

**Review by BMItr — APPROVED** [review]

**Review by chiljamgossow — APPROVED** [review]
LGTM

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/CertificatesViewModel.kt (hunk 1 above)
Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/CertificatesViewModel.kt:56]: I'll make it better in another PR :heart:

**Pull request merged**